wp-content/plugins/core/src/Templates/Controllers/Page.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+
+namespace Tribe\Project\Templates\Controllers;
+
+use Tribe\Project\Templates\Abstract_Template;
+use Tribe\Project\Templates\Component_Factory;
+use Tribe\Project\Templates\Components\Breadcrumbs;
+use Tribe\Project\Templates\Components\Button;
+use Tribe\Project\Templates\Components\Image;
+use Tribe\Project\Templates\Components\Pagination;
+use Tribe\Project\Templates\Controllers\Content\Header\Subheader;
+use Tribe\Project\Templates\Controllers\Sidebar\Main_Sidebar;
+use Twig\Environment;
+
+class Page extends Abstract_Template {
+	protected $path = 'page.twig';
+
+	/**
+	 * @var Header
+	 */
+	private $header;
+	/**
+	 * @var Subheader
+	 */
+	private $subheader;
+	/**
+	 * @var Main_Sidebar
+	 */
+	private $sidebar;
+	/**
+	 * @var Footer
+	 */
+	private $footer;
+
+	public function __construct(
```

Comment:
I _really_ like this change - injecting the other controllers as dependencies is a substantial improvement, and with our DI autowiring it's super-clean. :standing-ovation: